contrib/openssl-cmake/CMakeLists.txt

```diff
@@ -112,6 +112,7 @@ if(ARCH_AMD64)
         set(OPENSSL_SYSTEM "macosx")
     else()
         macro(perl_generate_asm FILE_IN FILE_OUT)
+            execute_process(COMMAND bash -c "mkdir -p $(dirname ${FILE_OUT})")
```

Comment:
It looks ok, but I don't understand why `bash -c` is needed.

Reply:
Copilot helped me write the codes😂.

Reply:
I just removed "bash -c" and had a try, I found "bash -c" is necessary.